framework/test/unit/application.spec.ts

```diff
@@ -194,6 +232,29 @@ describe('Application', () => {
 		});
 	});
 
+	describe('#registerModule', () => {
+		it('should throw error when module with same name is already registered', () => {
```

Comment:
What you probably meant here was:

_should throw error when PLUGIN with same name is already registered_